### ISSUE TITLE
Broken in HomeAssistant core-2022.3.0b1 & b2

### ISSUE THREAD

**Issue by coreGreenberet** [JoDehli/PyLoxone#101]
after upgrading homeassistant to core-2022.3.0b1 the PyLoxone integration has stopped working.

```
Logger: homeassistant.config_entries
Source: custom_components/loxone/api.py:77
Integration: PyLoxone ([documentation](https://github.com/JoDehli/PyLoxone), [issues](https://github.com/JoDehli/PyLoxone/issues))
First occurred: 13:51:52 (1 occurrences)
Last logged: 13:51:52
Error setting up entry PyLoxone for loxone


Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/requests_async/api.py", line 6, in request
    return await session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/requests_async/sessions.py", line 79, in request
    resp = await self.send(prep, **send_kwargs)
  File "/usr/local/lib/python3.9/site-packages/requests_async/sessions.py", line 136, in send
    r = await adapter.send(request, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/requests_async/adapters.py", line 42, in send
    ssl = httpcore.SSLConfig(cert=cert, verify=verify)
AttributeError: module 'httpcore' has no attribute 'SSLConfig'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 335, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/config/custom_components/loxone/__init__.py", line 141, in async_setup_entry
    if not await miniserver.async_setup():
  File "/config/custom_components/loxone/miniserver.py", line 110, in async_setup
    request_code = await self.lox_config.getJson()
  File "/config/custom_components/loxone/api.py", line 77, in getJson
    api_resp = await requests.get(
  File "/usr/local/lib/python3.9/site-packages/requests_async/api.py", line 11, in get
    return await request("get", url, params=params, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/requests_async/api.py", line 6, in request
    return await session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/requests_async/sessions.py", line 320, in __aexit__
    await self.close()
  File "/usr/local/lib/python3.9/site-packages/requests_async/sessions.py", line 311, in close
    await v.close()
  File "/usr/local/lib/python3.9/site-packages/requests_async/adapters.py", line 65, in close
    await self.pool.close()
TypeError: object NoneType can't be used in 'await' expression
```

**Comment by Bouni** [comment]
I have the same problem.
This custom integration uses [requests_async](https://github.com/encode/requests-async) which seems no longer active, actuall the repo was archived and the last release is from 2019.
The README even suggests `httpx` or `aiohttp` as alternatives.

As HA is using [aiohttp](https://github.com/home-assistant/core/blob/dev/requirements.txt) as a core requirement I strongly suggest that you refactor this integration and make use of aiohttp!

**Comment by vauriga** [comment]
I've already fixed this by refactoring to httpx, I will clean up and do a PR for it soon, maybe I will take a look at aiohttp

**Comment by JoDehli** [comment]
I already wrote this integration completely new and it depends completely on aiohttp. I separated the communication to the loxone from the home assistant stuff. It is a complete new. One of the next releases I will push this code. I leave the old versions untouched to if there is a big issue you can go back.

**Comment by vauriga** [comment]
@JoDehli great work! Push what you have and I will try to help.

**Comment by Bouni** [comment]
I'm also in for testing and help if you need it!

@JoDehli I would really appreciate if you could push your rewrite as a new branch so we can test!

**Comment by jdenver13** [comment]
Getting this error on updating to 2022.03.

**Comment by JoDehli** [comment]
Should be fixed in the master branch. I push a new release the next days.

**Comment by Bouni** [comment]
I've just installed the master branch manually and can confirm that it works again!
@JoDehli Do you replace httpx with aiohttp anyway?

**Comment by JoDehli** [comment]
I want to fix one more issue on the light controller then I push the new code to a separate branch. I think we can replace it there. Hope I have time on the weekend.

**Comment by JoDehli** [comment]
Fixed in the new 0.3.9 Release

**Comment by JoDehli** [comment]
Hey @Bouni and @vauriga, 

as I mentioned I made a big change on the Pyloxone. Main goal was to separate the websocket communication from the homeassistant stuff. For now I included it into the loxone folder as a sub-directory but for the future I want it completely separate.

If you want to help me especially for testing you can find the Code unter a separate git repo. Here is the link: 
https://github.com/JoDehli/PyLoxone_beta

I made a litte todo on readme main page. It would be great if you can help me. Thanks in advance! 

Jojo
